Description: Fix build on Hurd systems
 Hurd doesn't have PATH_MAX constant defined. Include externals.h before
 insternals.h in unix.c in order to fall back to hard-coded value (4096).
Author: Sam Protsenko <joe.skb7@gmail.com>
Bug-Debian: https://bugs.debian.org/873542
Reviewed-By: Peter Kosyh <p.kosyh@gmail.com>
Last-Update: 2017-08-28

--- instead-3.1.2.orig/src/unix.c
+++ instead-3.1.2/src/unix.c
@@ -34,6 +34,7 @@
 #include <stdlib.h>
 #include <time.h>
 
+#include "externals.h"
 #include "internals.h"
 
 #ifdef _USE_GTK
